skrf/vectorFitting.py

```diff
@@ -1,3 +1,4 @@
+from line_profiler import line_profiler
```

Comment:
tests fail because of this import. I guess this is for testing purpose, so can be removed
```suggestion
```